cmd/cluster/openstack/create_test.go

```diff
@@ -130,3 +131,178 @@ func TestCreateCluster(t *testing.T) {
 		})
 	}
 }
+
+func TestExtractCloud(t *testing.T) {
```

Comment:
It's possible here to use a test struct which could cover multiple cases? 
This is an example: https://github.com/openshift/hypershift/blob/main/api/util/configrefs/refs_test.go#L36

- This will make the readability better
- Reduce duplicated code
- Better organization.

Reply:
I explicitly avoided using a table-driven test here since there wasn't enough overlap between test cases. The resulting test case was full of "single-use" conditional branches that ultimately made the test case much more difficult to read (readability, organization) without a corresponding decrease in LOC (duplication).